src/components/SearchBar/index.jsx

```diff
@@ -15,8 +15,13 @@ const SearchBar = () => {
           type="text"
           placeholder={t("resources.searchBar.inputPlaceholder")}
           maxLength="15"
+          value={searchKey}
+          onChange={(e) => setSearchKey(e.target.value)}
```

Comment:
I did not know we can do this with props!
NOICE!!!

Reply:
Actually I didn't know too. I tried and it worked XD